src/ProjectNeon/Assets/Z_Tests/Features/Effects/ArmorFlatTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using NUnit.Framework;
+
+
+public sealed class ArmorFlatTests
+{
+
+    private EffectData data = new EffectData { EffectType = EffectType.ArmorFlat, FloatAmount = new FloatReference(1) } ;
+
+    private Member performer = new Member(
+        1,
+        "Good Dummy One",
+        "Paladin",
+        TeamType.Party,
+        new StatAddends()
+    );
+
+    private Member target = new Member(
+        2,
+        "Good Dummy Two",
+        "Wooden Dummy",
+        TeamType.Party,
+        new StatAddends().With(StatType.Armor, 5)
+    );
+
+    [Test]
+    public void ArmorFlat_ApplyEffect()
+    {
+        AllEffects.Apply(data, performer, new MemberAsTarget(target));
+        Assert.AreEqual(
+            6,
```

Comment:
It would be more clear if all of the numbers are in the test, so that one doesn't have to look everywhere to find out why `6` is the right number. 

i.e. Avoid Magic Numbers.